### PR TITLE
Added ongoing access doc.

### PR DESCRIPTION
**Pull request by JacobKingDev**

### CODE REVIEW

File: docs/docsets/.docset/Contents/Resources/Documents/ongoing-access.html [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+    <script src="js/jazzy.js" defer></script>
+    
+    <script src="js/lunr.min.js" defer></script>
+    <script src="js/typeahead.jquery.js" defer></script>
+    <script src="js/jazzy.search.js" defer></script>
+  </head>
+  <body>
+
+
+    <a title="Ongoing Access  Reference"></a>
```
Comment (Melisa69) [inline docs/docsets/.docset/Contents/Resources/Documents/ongoing-access.html]: Double spacing between `Access` and `Reference

File: docs/docsets/.docset/Contents/Resources/Documents/ongoing-access.html [hunk 2]
```diff
@@ -0,0 +1,188 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Ongoing Access  Reference</title>
```
Comment (Melisa69) [inline docs/docsets/.docset/Contents/Resources/Documents/ongoing-access.html]: Double spacing between `Access` and `Reference`

File: docs/docsets/.docset/Contents/Resources/Documents/ongoing-access.html [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+            <img class="header-icon" src="img/dash.png"/>
+            Install in Dash
+          </a>
+        </p>
+    </header>
+
+    <p class="breadcrumbs">
+      <a class="breadcrumb" href="index.html"> Reference</a>
+      <img class="carat" src="img/carat.png" />
+      Ongoing Access  Reference
```
Comment (Melisa69) [inline docs/docsets/.docset/Contents/Resources/Documents/ongoing-access.html]: Same comment as above

File: docs/docsets/.docset/Contents/Resources/Documents/ongoing-access.html (hunk 1 above)
Comment (JacobKingDev) [reply docs/docsets/.docset/Contents/Resources/Documents/ongoing-access.html]: Looks like this is generated by jazzy as no such spaces exist in the source files - I wonder if the other HTML pages have this too.

**Pull request merged**